support-frontend/assets/components/forms/customFields/error.jsx

```diff
@@ -7,39 +7,24 @@ import React, { type Node } from 'react';
 import { type Option } from 'helpers/types/option';
 
 import './error.scss';
+import { InlineError } from '@guardian/src-inline-error';
 
 // ----- Types ----- //
 export type PropsForHoc = {
-  error: Option<string>,
+  error: Option<string> | Option<Node>,
 };
 
 type Props = PropsForHoc & {
-  htmlFor: Option<string>,
   children?: Option<Node>,
 };
 
 // ----- Component ----- //
 
-function Error({ error, htmlFor, children }: Props) {
-  const Element = htmlFor ? 'label' : 'div';
+function Error({ error, children }: Props) {
   return (
     <div className={error ? 'component-form-error' : null}>
-      {children && children}
-      <Element
-        aria-hidden={!error}
```

Comment:
I'm assuming that the removal of the aria tags here is okay since these messages aren't on that page at all unless there's an error, therefore aren't going to confuse a screen-reader user.

Reply:
I've discussed this with Simon A and we believe it is fine, he is doing some more investigation and I'll update if needed.